ambiguous std::abs when build cpp on ubuntu1604

## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?
If possible, provide a recipe for reproducing the error.

Run `Build Cpp` on ubuntu1604(docker).

2. What did you expect to see?

Build succeed.

4. What did you see instead?

Build failed with the following error.

```
In file included from /root/incubator-pegasus/src/utils/output_utils.cpp:18:0:
/root/incubator-pegasus/src/utils/output_utils.h: In member function 'std::__cxx11::string dsn::utils::table_printer::to_string(T) [with T = double; std::__cxx11::string = std::__cxx11::basic_string<char>]':
/root/incubator-pegasus/src/utils/output_utils.h:180:22: error: call of overloaded 'abs(double&)' is ambiguous
     if (std::abs(data) < 1e-6) {
                      ^
In file included from /usr/include/c++/5/cstdlib:72:0,
                 from /root/incubator-pegasus/thirdparty/output/include/rapidjson/rapidjson.h:39,
                 from /root/incubator-pegasus/thirdparty/output/include/rapidjson/stream.h:15,
                 from /root/incubator-pegasus/thirdparty/output/include/rapidjson/ostreamwrapper.h:18,
                 from /root/incubator-pegasus/src/utils/output_utils.h:20,
                 from /root/incubator-pegasus/src/utils/output_utils.cpp:18:
```

6. What version of Pegasus are you using?
master

--- a/src/utils/output_utils.h
+++ b/src/utils/output_utils.h
@@ -20,7 +20,7 @@
 // IWYU pragma: no_include <bits/std_abs.h>
 #include <rapidjson/ostreamwrapper.h>
 #include <stdlib.h>
-#include <cmath>
+#include <cmath> // IWYU pragma: keep
 #include <iomanip>
 // IWYU pragma: no_include <new>
 #include <sstream> // IWYU pragma: keep

--- a/src/utils/output_utils.h
+++ b/src/utils/output_utils.h
@@ -20,6 +20,7 @@
 // IWYU pragma: no_include <bits/std_abs.h>
 #include <rapidjson/ostreamwrapper.h>
 #include <stdlib.h>
+#include <cmath>
 #include <iomanip>
 // IWYU pragma: no_include <new>
 #include <sstream> // IWYU pragma: keep

